llvm/lib/Target/RISCV/RISCVISelLowering.h

```diff
@@ -1015,6 +1022,8 @@ class RISCVTargetLowering : public TargetLowering {
 
   SDValue lowerVectorStrictFSetcc(SDValue Op, SelectionDAG &DAG) const;
 
+  SDValue LowerDYNAMIC_STACKALLOC(SDValue Op, SelectionDAG &DAG) const;
```

Comment:
Function names should start with a lower case letter. There's a long history Lower functions violating this rule and the bad naming keeps getting copied.